www/css/core.css

```diff
@@ -1818,6 +1818,7 @@ label.select > span + span::after{
 	margin-top: -3px; /* Adjust for Crimson Pro line-height */
 	text-shadow: 1px 1px 0 rgba(255, 255, 255, .5);
 	transition: color .5s;
+	z-index: -1;
```

Comment:
Why is the z-index changed?

Reply:
Without this change clicking exactly on the down arrow eats the click and doesn't activate the `<select>` element.

You can reproduce this today on:

https://standardebooks.org/ebooks
https://standardebooks.org/artworks

by clicking directly on the down arrow:

<img width="338" alt="Screenshot 2024-01-23 at 1 30 15 PM" src="https://github.com/standardebooks/web/assets/136965/16233ed9-d5f4-4d8e-915b-9ef5a0828d37">